src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -293,6 +301,10 @@ export const ProjectOverview: FC = () => {
           )}
         </div>
       )}
+      <UpdateProjectStepDialog
```

Comment:
I think it makes more sense to put this next to the DataButton. Dialogs use a Portal so it doesn't matter where they go in JSX.

Reply:
I was thinking of using this dialog for the update dates and location too

Reply:
It's called UpdateStep though?

Reply:
I renamed it `UpdateProjectDialog`